internal/captain/command.go

```diff
@@ -231,27 +234,12 @@ func (c *Command) AddLegacyChildren(children ...cobraCommander) {
 	}
 }
 
-func (c *Command) Find(args []string) *Command {
-	var innerFind func(*Command, []string) (*Command, []string)
-
-	innerFind = func(innerCmd *Command, innerArgs []string) (*Command, []string) {
-		if len(innerArgs) == 0 {
-			return innerCmd, innerArgs
-		}
-
-		cmd := innerCmd.findNext(innerArgs[0])
-		if cmd != nil {
-			return innerFind(cmd, innerArgs[1:])
-		}
-		return innerCmd, innerArgs
-	}
-
-	if len(args) < 2 {
-		return c
+func (c *Command) Find(args []string) (*Command, error) {
+	foundCobra, _, err := c.cobra.Find(args)
+	if err != nil {
+		return nil, err
```

Comment:
Wrap

Reply:
Why'd you remove this?

Reply:
Because of this https://github.com/ActiveState/cli/pull/1004#discussion_r500563763. Specifically:
> But this probably shouldn't be a localized error at all.

Is that not what you meant?

Reply:
That doesn't mean it shouldn't be wrapped, it just means to use `errs.Wrap` instead of `locale.WrapError`